pkg/kds/mux/client.go

```diff
@@ -128,6 +128,7 @@ func (c *client) startKDSMultiplex(ctx context.Context, log logr.Logger, conn *g
 		return
 	}
 	bufferSize := len(registry.Global().ObjectTypes())
+	// nolint:contextcheck
```

Comment:
Yeah, I think this comes because we break the "don't put context in a struct" rule in a lot of places